canonical/loki-worker-k8s-operator
# First Loki worker implementation

This is the first implementation of Loki worker charm for the HA deployment.

It's not really functional yet, since we need to finish the Loki coordinator charm.

# How to test it

1. Pack the charm: `charmcraft pack`
2. Deploy the charm: `juju deploy ./*.charm loki-worker --resource loki-image=docker.io/ubuntu/loki:3.0-22.04 --trust --config all=True`
3. Check the charm is in `blocked` state with the message: `Missing loki-cluster relation to a loki-coordinator charm`
